### ISSUE TITLE
confusion autour du nom "Fonofone" en haut à gauche

### ISSUE THREAD

**Issue by yves-fonofone** [hub-0-1/fonofone#135]
Comme il est en noir, ce qui se dégage c'est le cercle rouge dans la lettre o. On peut confondre avec le bouton d'enregistrement.

**Comment by OlivierRoss** [comment]
Il faudrait simplement que tu me fournisses une image / icône qui serait plus satisfaisante. Sinon, je peux mettre l'arrière-plan en blanc, mais ce ne serait pas très esthétique ...

**Comment by yves-fonofone** [comment]
En fait, il n'y a pas de raison de le laisser. C'est très récemment que j'ai remarqué sa présence.

**Comment by OlivierRoss** [comment]
C'est le bouton qui permet de sélectionner le Fonofone par défaut. Je pense que ce serait donc bien de le laisser. Il faudrait simplement réfléchir au UI / UX qui l'entoure.

**Comment by yves-fonofone** [comment]
Je pige pas. J’Ai beau cliquer dessus, il ne sa passe rien.

On en discutera dans notre rencontre avec François, mercredi ou jeudi midi, à ton choix.

Yves

_____________

www.fonofone.org <http://www.fonofone.org/>
https://www.electrocd.com/resultats/daoust
 <https://www.electrocd.com/resultats/daoust>

**Comment by OlivierRoss** [comment]
Explication plus complète : quand on créé une nouvelle zone, il y a un Fonofone par défaut qui est chargé. Pour éviter de charger n'importe que Fonofone @artificiel m'avait demandé de pouvoir définir le Fonofone par défaut. Donc, effectivement le bouton ne fait "rien" (de graphique), mais il définit le Fonofone utilisé pour les nouvelles zones.

**Comment by yves-fonofone** [comment]
Ok, je comprends. Alors il pourrait être invisible...

_____________

www.fonofone.org <http://www.fonofone.org/>
https://www.electrocd.com/resultats/daoust
 <https://www.electrocd.com/resultats/daoust>

**Comment by OlivierRoss** [comment]
Oui, mais dans ce cas, il sera difficile de cliquer dessus. On en parle avec François jeudi
Le 12 juin 2021 20:36 -0400, yves-fonofone ***@***.***>, a écrit :

Ok, je comprends. Alors il pourrait être invisible...

_____________

www.fonofone.org <http://www.fonofone.org/>
https://www.electrocd.com/resultats/daoust
<https://www.electrocd.com/resultats/daoust>
